sdcm/cluster.py

```diff
@@ -3649,7 +3649,11 @@ def update_scylla_packages(node, _queue):
             node.remoter.send_files(new_scylla_bin, '/tmp/scylla', verbose=True)
 
             self._wait_for_preinstalled_scylla(node)
+            logging.info("unzipping any tar.gz rpms")
+            node.remoter.run('tar -xvf /tmp/scylla/*.tar.gz -C /tmp/scylla/', ignore_status=True, verbose=True)
```

Comment:
1. can we add an existing check before compressing?
    - test -e /tmp/scylla/*.tar.gz 
2. It requires the rpm packages in the top directory, currently we won't install the rpms in the sub-directory of /tmp/scylla/

Reply:
better to check that the files exist before untaring

Reply:
no, it doesn't really matter like we don't check that the rpms are there.